templates/.github/ISSUE_TEMPLATE.md

```diff
@@ -1,7 +1,35 @@
-<!--
-1. Check the version of package you are using. If it's not the newest version, update and try again (see changelog while updating!).
-2. If the issue is still there, write a minimal project showing the problem and expected output.
-3. Link to the project and mention Node version and OS in your report.
+> Briefly describe the issue you are experiencing (or the feature you want to see
+added to the plugin). Tell us what you were trying to do and what happened
+instead. Remember, this is _not_ a place to ask questions. For that, go to
```

Comment:
but it is a place to ask questions - when reporting bugs and issues. be more specific about this not being a forum for _support_ questions.

Reply:
Yup. Emphasize that support should go elsewhere. This needs a wording change.